### DISCUSSION TITLE
테스트 서버에서 회원가입 진행 불가

### DISCUSSION THREAD

**Discussion by Dae-Hwa** [Sc0up/scoup-backend#181] [Troubleshooting docs]
인터셉터에서 예외 처리되는 것으로 추정

헤로쿠 문제인지, 코드 문제인지 파악 필요

**Comment by janeljs** [comment]
AWS로 배포시 정상 작동하는 것 확인하였습니다!

**Comment by Dae-Hwa** [comment]
헤로쿠
https://sc0up.herokuapp.com/api/

**Reply by Dae-Hwa** [reply]
이메일 회원가입 정상 작동

**Reply by Dae-Hwa** [reply]
이메일 로그인 정상 작동

**Reply by Dae-Hwa** [reply]
이메일 로그인 후 스케줄 후보 정상 작동

**Reply by janeljs** [reply]
GitHub 사용자 정보 요청 정상 작동

**Reply by janeljs** [reply]
Kakao 사용자 정보 요청 정상 작동

**Reply by janeljs** [reply]
Google 사용자 정보 요청 정상 작동

**Reply by janeljs** [reply]
로그인 후 그룹 생성 정상 작동

**Reply by Dae-Hwa** [reply]
추후 문제 발견되면 다시 재개
